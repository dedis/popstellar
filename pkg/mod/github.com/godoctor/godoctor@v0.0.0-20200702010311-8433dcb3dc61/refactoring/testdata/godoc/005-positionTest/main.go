package main // <<<<< godoc,1,1,1,1,pass

import "fmt"

func main() {
	Exported()
}

func Exported() {
  fmt.Println("Hello, Go")
}

type Shaper interface {

}

type Rectangle struct {

}
