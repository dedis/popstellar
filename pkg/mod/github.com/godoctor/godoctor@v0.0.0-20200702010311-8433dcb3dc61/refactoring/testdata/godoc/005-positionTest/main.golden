package main // <<<<< godoc,1,1,1,1,pass

import "fmt"

func main() {
	Exported()
}

// Exported TODO: NEEDS COMMENT INFO
func Exported() {
	fmt.Println("Hello, Go")
}

// Shaper TODO: NEEDS COMMENT INFO
type Shaper interface {
}

// Rectangle TODO: NEEDS COMMENT INFO
type Rectangle struct {
}
